diff --git a/Util/BuildTools/BuildPythonAPI.bat b/Util/BuildTools/BuildPythonAPI.bat
index 1234567..abcdefg 100644
--- a/Util/BuildTools/BuildPythonAPI.bat
+++ b/Util/BuildTools/BuildPythonAPI.bat
@@ -93,10 +93,10 @@ cd "%PYTHON_LIB_PATH%"
 rem if exist "%PYTHON_LIB_PATH%dist" goto already_installed
 
 rem ============================================================================
-rem -- Check for py ------------------------------------------------------------
+rem -- Check for python --------------------------------------------------------
 rem ============================================================================
 
-where py 1>nul
+where python 1>nul
 if %errorlevel% neq 0 goto error_py
 
 rem Build for Python 2
@@ -109,7 +109,7 @@ rem Build for Python 3
 rem
 if %BUILD_FOR_PYTHON3%==true (
     echo Building Python API for Python 3.
-    py -3 setup.py bdist_egg bdist_wheel
+    python setup.py bdist_egg bdist_wheel
     if %errorlevel% neq 0 goto error_build_wheel
 )
 
@@ -136,11 +136,9 @@ goto success
 
 :error_py
     echo.
-    echo %FILE_N% [ERROR] An error ocurred while executing the py.
+    echo %FILE_N% [ERROR] An error ocurred while executing python.
     echo %FILE_N% [ERROR] Possible causes:
-    echo %FILE_N% [ERROR]  - Make sure "py" is installed.
-    echo %FILE_N% [ERROR]  - py = python launcher. This utility is bundled with Python installation but not installed by default.
-    echo %FILE_N% [ERROR]  - Make sure it is available on your Windows "py".
+    echo %FILE_N% [ERROR]  - Make sure "python" is installed and available on your Windows PATH.
     goto bad_exit
 
 :error_build_wheel
diff --git a/Util/BuildTools/Package.bat b/Util/BuildTools/Package.bat
index 2345678..bcdefgh 100644
--- a/Util/BuildTools/Package.bat
+++ b/Util/BuildTools/Package.bat
@@ -119,10 +119,10 @@ set BUILD_FOLDER=%INSTALLATION_DIR:/=\%UE4Carla\
 if %DO_PACKAGE%==true (
 
     if %USE_CARSIM% == true (
-        py -3 %ROOT_PATH%Util/BuildTools/enable_carsim_to_uproject.py -f="%ROOT_PATH%Unreal/CarlaUE4/CarlaUE4.uproject" -e
+        python %ROOT_PATH%Util/BuildTools/enable_carsim_to_uproject.py -f="%ROOT_PATH%Unreal/CarlaUE4/CarlaUE4.uproject" -e
         echo CarSim ON > "%ROOT_PATH%Unreal/CarlaUE4/Config/CarSimConfig.ini"
     ) else (
-        py -3 %ROOT_PATH%Util/BuildTools/enable_carsim_to_uproject.py -f="%ROOT_PATH%Unreal/CarlaUE4/CarlaUE4.uproject"
+        python %ROOT_PATH%Util/BuildTools/enable_carsim_to_uproject.py -f="%ROOT_PATH%Unreal/CarlaUE4/CarlaUE4.uproject"
         echo CarSim OFF > "%ROOT_PATH%Unreal/CarlaUE4/Config/CarSimConfig.ini"
     )
 
diff --git a/Util/BuildTools/BuildCarlaUE4.bat b/Util/BuildTools/BuildCarlaUE4.bat
index 3456789..cdefghi 100644
--- a/Util/BuildTools/BuildCarlaUE4.bat
+++ b/Util/BuildTools/BuildCarlaUE4.bat
@@ -155,10 +155,10 @@ if %USE_OMNIVERSE% == true (
 )
 
 if %USE_CARSIM% == true (
-    py -3 %ROOT_PATH%Util/BuildTools/enable_carsim_to_uproject.py -f="%ROOT_PATH%Unreal/CarlaUE4/CarlaUE4.uproject" -e
+    python %ROOT_PATH%Util/BuildTools/enable_carsim_to_uproject.py -f="%ROOT_PATH%Unreal/CarlaUE4/CarlaUE4.uproject" -e
     set CARSIM_STATE="CarSim ON"
 ) else (
-    py -3 %ROOT_PATH%Util/BuildTools/enable_carsim_to_uproject.py -f="%ROOT_PATH%Unreal/CarlaUE4/CarlaUE4.uproject"
+    python %ROOT_PATH%Util/BuildTools/enable_carsim_to_uproject.py -f="%ROOT_PATH%Unreal/CarlaUE4/CarlaUE4.uproject"
     set CARSIM_STATE="CarSim OFF"
 )
 if %USE_CHRONO% == true (
diff --git a/Util/BuildTools/BuildOSM2ODR.bat b/Util/BuildTools/BuildOSM2ODR.bat
index 4567890..defghij 100644
--- a/Util/BuildTools/BuildOSM2ODR.bat
+++ b/Util/BuildTools/BuildOSM2ODR.bat
@@ -70,7 +70,7 @@ rem -- Local Variables ---------------------------------------------------------
 rem Set the visual studio solution directory
 rem
 set OSM2ODR_VSPROJECT_PATH=%INSTALLATION_DIR:/=\%osm2odr-visualstudio\
-set OSM2ODR_SOURCE_PATH=%INSTALLATION_DIR:/=\%om2odr-source\
+set OSM2ODR_SOURCE_PATH=%INSTALLATION_DIR:/=\%osm2odr-source\
 set OSM2ODR_INSTALL_PATH=%ROOT_PATH:/=\%PythonAPI\carla\dependencies\
 set OSM2ODR__SERVER_INSTALL_PATH=%ROOT_PATH:/=\%Unreal\CarlaUE4\Plugins\Carla\CarlaDependencies
 set CARLA_DEPENDENCIES_FOLDER=%ROOT_PATH:/=\%Unreal\CarlaUE4\Plugins\Carla\CarlaDependencies\
@@ -114,7 +114,7 @@ if %BUILD_OSM2ODR% == true (
         set PLATFORM=
     )
 
-    cmake -G %GENERATOR%´-A x64^
+    cmake -G %GENERATOR% %PLATFORM%^
         -DCMAKE_CXX_FLAGS_RELEASE="/MD /MP"^
         -DCMAKE_INSTALL_PREFIX="%OSM2ODR_INSTALL_PATH:\=/%"^
         -DPROJ_INCLUDE_DIR=%INSTALLATION_DIR:/=\%\proj-install\include^
